model/registry/process.yaml

```
@@ -41,6 +41,18 @@ groups:
           The PID of the process's group leader. This is also the process group
           ID (PGID) of the process.
         examples: [23]
+      - id: process.executable.build_id.gnu
+        stability: experimental
+        type: string
+        brief: >
+          The GNU build ID as can be found in the `.note.gnu.build-id` ELF section.
+        examples: ['c89b11207f6479603b0d49bf291c092c2b719293']
+      - id: process.executable.build_id.go
```

`
```suggestion
      - id: process.executable.go.build_id
```

reverse namespace here too

---

see https://github.com/open-telemetry/semantic-conventions/pull/1329#discussion_r1723527952

---

Switched namespaces as suggested with https://github.com/open-telemetry/semantic-conventions/pull/1329/commits/d779b0d81f1354f230bcd93b67b52f32dee6cfa0.